diff --git a/Test.php b/Test.php
index 28b33a3..6209f0e 100644
--- a/Test.php
+++ b/Test.php
@@ -1,49 +1,54 @@
 <?php
 
-$l = array_Map(function ($s) {
+$l = array_map(function ($s) {
 	return 'b';
 }, [1,2,3]);
 
-print_r ($l );
+print_r($l);
 
 $f = TRUE;
-$f = true;
+$f = TRUE;
 
 $a = [];
 
-$foo -> $bar = 1;
+$foo->$bar = 1;
 
-$a = 1+2;
-$b= 'c';
-$b ='c';
-$b = 'c' ;
+$a = 1 + 2;
+$b = 'c';
+$b = 'c';
+$b = 'c';
 
-$c = (int) $b;
 $c = (int)$b;
-$c = (int )$b;
+$c = (int)$b;
+$c = (int)$b;
 
-for ($i=0;$i<10;$i++) {
+for ($i = 0; $i < 10; $i++) {
 	echo $i.'abc';
 	echo $i,'abc';
 }
 
-function abc1($a=1, $b=2) {
+function abc1($a = 1, $b = 2)
+{
 	echo 'def';
 }
 
-function abc2($a = 1, $b = 2) {
+function abc2($a = 1, $b = 2)
+{
 	echo 'def';
 }
 
-function abc3( $a = 1) {
+function abc3($a = 1)
+{
 	echo 'def';
 }
 
-function abc4( ) {
+function abc4()
+{
 	echo 'def';
 }
 
-function abc5(){
+function abc5()
+{
 	echo 'def';
 }
 
@@ -52,25 +57,27 @@ function abc6()
 	echo 'def';
 }
 
-class TestClassA {
+class TestClassA
+{
 
 }
 
 class TestClassB
 {
 
-	public function abc() {
+	public function abc()
+	{
 		if ($a > 1) {
 			echo 'yes';
 		}
-		if($a>1) {
+		if ($a > 1) {
 			echo 'yes';
 		}
-		if($a   >   1) {
+		if ($a > 1) {
 			echo 'yes';
 		}
 
-		$s=new     StdClass();
+		$s = new StdClass();
 	}
 }
 
